transforms/code/license_check/ray/pyproject.toml

```diff
@@ -0,0 +1,49 @@
+[project]
+name = "license_check_transform_ray"
```

Comment:
To match conventions, this should be named dpk_license_check_transform_ray

Reply:
done